wallet-webapps/src/main/webapp/vue-app/wallet-overview/components/WalletOverview.vue

```diff
@@ -120,6 +123,25 @@ export default {
           this.$root.$emit('application-loaded');
         });
     },
+    countRewards() {
+      return fetch('/portal/rest/wallet/api/reward/countRewards', {
+        method: 'GET',
+        credentials: 'include',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+      }).then((resp) => {
+        if (resp && resp.ok) {
+          return resp.json();
+        } else {
+          throw new Error('Error count rewards');
+        }
+      }).catch((error) => {
+        console.error(error);
+        this.error = this.$t('exoplatform.wallet.error.errorCountRewards');
+      });
+    },
```

Comment:
Could you export this function in a dedicated JS API file for rewarding

Reply:
done !